DESCRIPTION

```diff
@@ -1,11 +1,11 @@
 Package: phers
 Type: Package
 Title: Calculate phenotype risk scores 
-Version: 0.0.0.9000
-Authors@R: as.person(c(
-  "Jake Hughey <jakejhughey@gmail.com> [aut, cre]",
-  "Layla Aref <larefnezhad@gmail.com> [aut]"))
-Description: More about what it does.
+Version: 0.0.1
+Authors@R: c(
+  person('Jake', 'Hughey', , 'jakejhughey@gmail.com', c('aut', 'cre')),
+  person('Layla', 'Aref', , 'larefnezhad@gmail.com', 'aut'))
+Description: Use phenotype risk scores based on linked clinical and genetic data to study Mendelian disease and rare genetic variants.
```

Comment:
@jakejh do we want to add the DOI for the first phers paper?

Reply:
Good call.